GameOfLife/src/App.java

```diff
@@ -1,5 +1,63 @@
-public class App {
-    public static void main(String[] args) throws Exception {
-        System.out.println("Hello, World!");
-    }
-}
+package src;
+
+import java.awt.BorderLayout;
+import java.awt.Container;
+import java.awt.GridLayout;
+
+import javax.swing.JButton;
+import javax.swing.JFrame;
+import javax.swing.JPanel;
+
+import View.Cell;
+
+public class App extends JFrame {
+	private JFrame frame;
+	Cell[][] buttons = new Cell[25][25];
```

Comment:
Currently the model is at 15x15, we will eventually put the board dimensions into a configuration class